composite/proof-of-reserves/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "src",
+    "typeRoots": ["../../node_modules/@types", "../../typings", "./typings"]
```

Comment:
Do we need this?

Reply:
Yes, as we have `typings` package with types for `'@chainlink/ea-bootstrap'` and `'@chainlink/external-adapter'` as they are still in JS-land. The other two definitions are just here as a template.